src/solvers/bitwuzla/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+set(Bitwuzla_MIN_VERSION "0.1")
+
+if(DEFINED Bitwuzla_DIR)
+    set(ENABLE_BITWUZLA ON)
+endif()
+
+if(EXISTS $ENV{HOME}/bitwuzla)
+    set(ENABLE_BITWUZLA ON)
+endif()
+
+if(ENABLE_BITWUZLA)
+    find_package(Bitwuzla REQUIRED
+            HINTS ${Bitwuzla_DIR}/lib/cmake $ENV{HOME}/bitwuzla)
+
+    message(STATUS "Found Bitwuzla at: ${Bitwuzla_DIR}")
+    message(STATUS "Bitwuzla version: ${Bitwuzla_VERSION}")
+    if(Bitwuzla_VERSION VERSION_LESS Bitwuzla_MIN_VERSION)
+        message(FATAL_ERROR "Expected version ${Bitwuzla_MIN_VERSION} or greater")
+    endif()
+   
+    add_library(solverbitw bitwuzla_conv.cpp)
+    target_include_directories(solverbitw
+           # PRIVATE ${Boost_INCLUDE_DIRS}
+            PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
+    # We cannot link both Bitwuzla::bitwuzla and Boolector::boolector at the moment
+    # due to "multiple definition" errors
+    target_link_libraries(solverbitw Bitwuzla::bitwuzla)
```

Comment:
@rafaelsamenezes can you help them out here?

Supporting bitwuzla means dropping support for boolector?